### DISCUSSION TITLE
treesitter-rewrite Extra

### DISCUSSION THREAD

**Discussion by dpetka2001** [LazyVim/LazyVim#3214] [General]
@folke This is really a nitpick, but would it be possible to just  comment out everything in `lazyvim.plugins.extras.ui.treesitter-rewrite` and also comment out the corresponding part from `lazyvim.util.plugin.deprecated_extras` so that it doesn't show an error each time we invoke the `:LazyExtras` UI? This way it won't even parse the spec and it also won't appear under `LazyExtras`. Didn't want to do a PR for this, as you might have your own reasons and thought it'd be best to ask you first.

**Comment by folke** [comment]
Just removed it. You will still need to remove it from `lazyvim.json` manually if you had it in there at some point

**Reply by dpetka2001** [reply]
I never had it in `lazyvim.json` but the error still popped up.
